### PR TITLE
feat: add organization custom attributes

### PR DESCRIPTION
**Pull request by ISulimanov**
Fix: https://github.com/casdoor/casdoor/issues/1767

ezgif-5-517a58f57e

### CODE REVIEW

File: object/migrator.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ func DoMigration() {
 		&Migrator_1_101_0_PR_1083{},
 		&Migrator_1_235_0_PR_1530{},
 		&Migrator_1_240_0_PR_1539{},
+		&Migrator_1_Add_Orgainzation_Prop{},
```
Comment (hsluoyz) [inline object/migrator.go:29]: Naming is wrong

File: web/src/locales/de/data.json [hunk 2]
```diff
@@ -380,6 +380,8 @@
     "Is profile public - Tooltip": "Nach der Schließung können nur globale Administratoren oder Benutzer in der gleichen Organisation auf die Profilseite des Benutzers zugreifen",
     "Modify rule": "Regel ändern",
     "New Organization": "Neue Organisation",
+    "Properties": "Eigenschaften",
```
Comment (hsluoyz) [inline web/src/locales/de/data.json:383]: reuse

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: object/migrator.go (hunk 1 above)
Comment (aegoroff) [reply object/migrator.go:29]: Should i name it Migrator_1_301_0_PR_1768 or something other name? I couldn't find any instructions about migrations naming.

File: web/src/locales/de/data.json (hunk 2 above)
Comment (aegoroff) [reply web/src/locales/de/data.json]: done

File: object/migrator.go (hunk 1 above)
Comment (hsluoyz) [reply object/migrator.go]: @aegoroff yes, that's it

File: object/migrator_1_301_0_PR_1768.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+func (*Migrator_1_301_0_PR_1768) IsMigrationNeeded() bool {
+	exist, _ := adapter.Engine.IsTableExist("organization")
+	return exist
+}
+
+func (*Migrator_1_301_0_PR_1768) DoMigration() *migrate.Migration {
+	migration := migrate.Migration{
+		ID: "20230420MigrateOrganization--Create a new field 'properties' for table `organization`",
+		Migrate: func(engine *xorm.Engine) error {
+			return engine.Sync2(&Organization{})
```
Comment (hsluoyz) [inline object/migrator_1_301_0_PR_1768.go:33]: Casdoor runs Sync2 in every start. Is this migrator really needed?

File: web/src/locales/es/data.json [hunk 4]
```diff
@@ -380,6 +381,7 @@
     "Is profile public - Tooltip": "Después de estar cerrado, solo los administradores globales o usuarios de la misma organización pueden acceder a la página de perfil del usuario",
     "Modify rule": "Modificar regla",
     "New Organization": "Nueva organización",
+    "Properties - Tooltip": "Propiedades de la organización",
```
Comment (hsluoyz) [inline web/src/locales/es/data.json:384]: Reuse `Properties - Tooltip`, you should not add any new words

File: web/src/locales/en/data.json [hunk 5]
```diff
@@ -380,6 +381,7 @@
     "Is profile public - Tooltip": "After being closed, only global administrators or users in the same organization can access the user's profile page",
     "Modify rule": "Modify rule",
     "New Organization": "New Organization",
+    "Properties - Tooltip": "Properties of the organization",
```
Comment (hsluoyz) [inline web/src/locales/en/data.json:384]: Modify translated words to `Properties of the object` to cover both meaning

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: web/src/locales/en/data.json (hunk 5 above)
Comment (aegoroff) [reply web/src/locales/en/data.json:384]: okay

File: web/src/locales/es/data.json (hunk 4 above)
Comment (aegoroff) [reply web/src/locales/es/data.json]: done

File: web/src/locales/en/data.json (hunk 5 above)
Comment (aegoroff) [reply web/src/locales/en/data.json]: done

**Comment by hsluoyz** [comment]
I don't think this feature is super useful for now. You can extend by yourself if you need more fields to store data. Closed here

**Pull request closed without merging**